### DISCUSSION TITLE
v7.2.5 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#17555] [General]
### Engine Updates and Fixes

- Fix native library loading for osx-arm64 (#17495) (Thanks @awakecoding!)

### Tests

- Make Assembly Load Native test work on a FX Dependent Linux Install (#17496)
- Enable more tests to be run in a container. (#17294)
- Switch to using GitHub action to verify markdown links for PRs (#17281)
- Try to stabilize a few tests that fail intermittently (#17426)
- TLS test fix back-port (#17424)

### Build and Packaging Improvements

<details>

<summary>

<p>Bump .NET SDK to 6.0.301 (Internal 21218)</p>

</summary>

<ul>
<li>Update Wix file (Internal 21242)</li>
<li>Conditionally add output argument</li>
<li>Rename mariner package to cm (#17506)</li>
<li>Backport test fixes for 7.2 (#17494)</li>
<li>Update dotnet-runtime version (#17472)</li>
<li>Update to use <code>windows-latest</code> as the build agent image (#17418)</li>
<li>Publish preview versions of mariner to preview repo (#17464)</li>
<li>Move <code>cgmanifest</code> generation to daily (#17258)</li>
<li>Fix mariner mappings (#17413)</li>
<li>Make sure we execute tests on LTS package for older LTS releases (#17430)</li>
<li>Add a finalize template which causes jobs with issues to fail (#17428)</li>
<li>Make mariner packages Framework dependent (#17425)</li>
<li>Base work for adding mariner amd64 package (#17417)</li>
</ul>

</details>

[7.2.5]: https://github.com/PowerShell/PowerShell/compare/v7.2.4...v7.2.5

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - B2F77E41916D16595FDCB27AB2CE94DBDBCE1C873E17BC4FDBC1EF9F719BE1D2
- powershell_7.2.5-1.deb_amd64.deb
  - 9132D97FEA77D5A8DE56B5801B7579BDC1CE669DAE084295DDAE6B43CF5A3FDB
- powershell-7.2.5-1.cm.x86_64.rpm
  - F6C437FA1ED959E26642AAECAFFFC174BE6D54F9B19E7432383C61E25BAFC15C
- powershell-7.2.5-1.rh.x86_64.rpm
  - D5ED557B70F0917EF95812D42F94AB660E2442E6C7B05D53C670EA16D1298F36
- powershell-7.2.5-linux-alpine-x64.tar.gz
  - EE832FD7C0D18DD2889D28FA7C1B7FB927CB015C60058A67F78858C5D69BCFE9
- powershell-7.2.5-linux-arm32.tar.gz
  - 30A3498FBAFD2FA6867F4C486F3630D2B9B42246B769DE2F950BF7218056DB05
- powershell-7.2.5-linux-arm64.tar.gz
  - 709265A0B99232CD8AB6F8A02C01F3AEE94262B959E1A4B0FD871C8789C07396
- powershell-7.2.5-linux-x64.tar.gz
  - A6CA50BC03A1F508B0177A9DDB63164D344A6B9CEE4CF4D459A04E72EA10C468
- powershell-7.2.5-linux-x64-fxdependent.tar.gz
  - 42B976F97F64D79EE771292D18DDF4326936361FA508E819B9BCA223920AE6BC
- powershell-7.2.5-osx-arm64.pkg
  - 2ABB2EA7CCACE636E2288AFF1D1C8BA1BF688F10839FEEA462988429DCC902FC
- powershell-7.2.5-osx-arm64.tar.gz
  - 74462A44A62FC2A6EF7D0D5430CBFA843AADC46FD59B2B5A25B495C583B5602D
- powershell-7.2.5-osx-x64.pkg
  - D007BC8C1E2BDB43E8B54CCACB86499A619588866255EAB5A823F46228587B2C
- powershell-7.2.5-osx-x64.tar.gz
  - 3DE73676EDAC53BFF1888AC91A9A3DC3D2BC1D322FF2AED7E8B4B148483DFE02
- PowerShell-7.2.5-win-arm32.zip
  - A37FA125A3508228458AEC36B61DDF62229FF0FD73FEB632B217B55972136099
- PowerShell-7.2.5-win-arm64.zip
  - 6CC428EBDBAD88EA94B75EAD18E23C6747C24B8AD9163F90A6EA35E91BB4CAA8
- PowerShell-7.2.5-win-fxdependent.zip
  - 77D6887AB5311B153C18A7D0327C23A7D89483EF2EF8D1BAADBDEFB47AD46034
- PowerShell-7.2.5-win-fxdependentWinDesktop.zip
  - 467D85BB15F7A7C9B9AE22C76AC3D093575B7B50780C5CADF3909584CEB1D91D
- PowerShell-7.2.5-win-x64.msi
  - 632A69EA46F966A03A1EE0AA1647C5970DF7B5936BDF27DF3CF18E6C63C21198
- PowerShell-7.2.5-win-x64.zip
  - B686207D13D728F44DF2BACB676D77EF4BD142A739A96798EBD85FDB5467866C
- PowerShell-7.2.5-win-x86.msi
  - 820C87D28425D7EC902730044C60C894E015FE4C899D61C427844FB31052D9C1
- PowerShell-7.2.5-win-x86.zip
  - E6F898E3C348A9DCBDDBC76787BF4031DBB68A5946DC6E9FD4310EE06B264CFF
- powershell-lts_7.2.5-1.deb_amd64.deb
  - D7F432866B4DA70A8E0ED7E2221C165406F2E04F2C691E69960CCF85AA53BA93
- powershell-lts-7.2.5-1.cm.x86_64.rpm
  - 8F93C52E075B7BB67FE27A69450AD2B994DE65B009F6D6C722202E0CFBEDBFD7
- powershell-lts-7.2.5-1.rh.x86_64.rpm
  - 4515C700044F65C9D9E25D29A31F522FC0FA3D7008F0BC2736E711D2C6B161ED
- powershell-lts-7.2.5-osx-arm64.pkg
  - 1F61B4F1B46F425C794FD73FE7D02D7FF9B63CA1088A46D0D7DA627B2F209E74
- powershell-lts-7.2.5-osx-x64.pkg
  - 032125F1DB6A4A3E583E2D87CC23C382A8462A9A8D753986B6EDA6CE3E50BD86

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.2.5'>v7.2.5 Release of PowerShell</a>.</em>

**Comment by yuguuk** [comment]
When is this update showing up on winget?

**Reply by lambdakilo** [reply]
Just now I did `winget search powershell` and the results included 7.2.5

**Comment by enderandpeter** [comment]
You killin' it, OGs.

**Comment by mahmoud-syr** [comment]
Man! I love the fact that you guys publish hashes with files ❤️ 
![image](https://user-images.githubusercontent.com/65822550/175890565-1435935b-7557-4cb4-b0c1-401825792d32.png)

One question though, why mac files have the "lts" added to the file name?

**Comment by aragornii** [comment]
I was wondering, usually after how much time the update is made available on Windows Update?

**Reply by aragornii** [reply]
Just updated through Windows Updates, so ∼10 days.

**Comment by ghost** [comment]
salut,
je suis sous Windows 11 et je suis apprenti développeur quel fichier je doit télécharger ?
